product/toolbars/containers_center_tb.yaml

```diff
@@ -52,6 +52,12 @@
     :title: Monitoring
     :text: Monitoring
     :items:
+    - :button: container_timeline
+      :image: timeline
+      :text: "Timelines"
+      :title: "Show Timelines for this Container"
+      :url: '/explorer'

```

Comment:
> @dkorn i think we don't need to set url here, as we are going thru x_button method

I tried removing it but it doesn't work. The progress spinner never stops and the timeline isn't loaded.

from development.log:
WARN -- : DEPRECATION WARNING: Returning a hash from a #scope or #default_scope block is deprecated. Please return an actual scope object instead. E.g. scope :red, -> { where(color: 'red') } rather than scope :red, -> { { conditions: { color: 'red' } } }. (The scope was defined at /home/dkorn/git/manageiq/app/models/miq_schedule.rb:22.). (called from call at /home/dkorn/.gem/ruby/gems/activerecord-deprecated_finders-1.0.4/lib/active_record/deprecated_finders/base.rb:39)